device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py

```diff
@@ -109,7 +111,95 @@ def get_low_power_mode(self, port_num):
         return True
 
     def set_low_power_mode(self, port_num, lpmode):
-        raise NotImplementedError
+        # Check for invalid port_num
+        if port_num < self.port_start or port_num > self.port_end:
+            return False
+
+        try:
+            reg_file = open(
+                "/sys/devices/platform/dx010_cpld/qsfp_lpmode", "r+")
```

Comment:
Please merge above 2 lines into one.

Reply:
Fixed in 5db212356dfce548f532d8398de43fed8dccf973